starter-repos/app-template/skaffold/render.yaml

```diff
@@ -36,6 +46,15 @@ skaffold-render:
     - cp /tmp/staging-west-deployment.yaml config/envs/staging-west/deployment.yaml
     - cp /tmp/prod-central-deployment.yaml config/envs/prod-central/deployment.yaml
     - cp /tmp/prod-east-deployment.yaml config/envs/prod-east/deployment.yaml
-  artifacts:
-    paths:
-      - config/envs
+
+    - |
+      # If files have changed, commit them back to the env repo in the staging branch
+      if [ -z "$(git status --porcelain)" ]; then
+        echo "No changes found in env repository."
+      else
+        git add config
+        git commit -m "update built manfiests"
```

Comment:
In the commit message, can you add the commit sha that triggers the current job?

Reply:
Done. it looks like this. https://screenshot.googleplex.com/6MQSeaK7v5TCnTZ.png